File: 100-Days-Series/Day 38/IntersectionOfTwoArrays.c
#include<stdio.h>
// Functiont to Find common elements between two arrays.
void findCommonElements(int arr1[], int size1, int arr2[], int size2) {
    printf("Common elements: ");
    for (int i = 0; i < size1; i++) {
        for (int j = 0; j < size2; j++) {
            if (arr1[i] == arr2[j]) {
                printf("%d ", arr1[i]);
                break; // Avoid duplicates
            }
        }
    }
    printf("\n");
}
// Let's Start here
int main() {
    int arr1[] = {1, 2, 3, 4, 5};
    int arr2[] = {3, 4, 5, 6, 7};
    int size1 = sizeof(arr1) / sizeof(arr1[0]);
    int size2 = sizeof(arr2) / sizeof(arr2[0]);
    findCommonElements(arr1, size1, arr2, size2);
}

File: 100-Days-Series/Day 38/IntersectionOfTwoArrays.java
public class IntersectionOfTwoArrays {
    // Functiont to Find common elements between two arrays.
    public static void findCommanElements(int arr1[],int arr2[]){
        for(int i=0;i<arr1.length;i++){
            for(int j=0;j<arr2.length;j++){
                if(arr1[i]==arr2[j]){
                    System.out.print(arr1[i]+" ");
                    break;
                }
            }
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start here
        int arr1[] = {1, 2, 3, 4, 5};
        int arr2[] = {3, 4, 5, 6, 7};
        findCommanElements(arr1, arr2);
    }
}
